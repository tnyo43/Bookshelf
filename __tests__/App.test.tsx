import React from "react";
import { create } from "react-test-renderer";
import { App } from "../src/components/App";

describe("ButtonComponent", () => {
  it("renderes correctly", () => {
    const root = create(<App />);
    expect(root.toJSON()).toMatchSnapshot();
  });
});
