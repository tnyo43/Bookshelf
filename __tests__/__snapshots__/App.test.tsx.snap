// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonComponent renderes correctly 1`] = `
<div>
  Hello World!
</div>
`;
